.gitignore

```diff
@@ -186,3 +186,5 @@ testem.log
 .DS_Store
 Thumbs.db
 *.vscode*
+gemini/sample-apps/finance-advisor-spanner/gcs-bucket.zip
+gemini/sample-apps/finance-advisor-spanner/.env
```

Comment:
```suggestion
```
Can you change the `.env_TMP` file to `.env` then remove this line?

Reply:
done